setup.py

```diff
@@ -28,14 +28,14 @@
 setup(
     name=NAME,
     version=VERSION,
-    description="Purity//FB REST Client",
+    description="Pure Storage FlashBlade REST 1.9 Python SDK",
     author_email="info@purestorage.com",
     url="",
     keywords=["Swagger", "Purity//FB REST Client"],
     install_requires=REQUIRES,
     packages=find_packages(),
     include_package_data=True,
     long_description="""\
-    Client for Purity//FB REST API (1.0 - 1.8), developed by [Pure Storage, Inc](http://www.purestorage.com/). Documentations can be found at [purity-fb.readthedocs.io](http://purity-fb.readthedocs.io/).
+    Pure Storage FlashBlade REST 1.9 Python SDK, developed by [Pure Storage, Inc](http://www.purestorage.com/). Documentations can be found at [purity-fb.readthedocs.io](http://purity-fb.readthedocs.io/).
```

Comment:
can we still note somewhere that this can be used for REST API versions (1.0 - 1.9)?

Reply:
This comment string gets stamped out at the top of every generated python file. Should I be updating all of them?

How about:

> Pure Storage FlashBlade REST 1.9 Python SDK. Compatible with REST API versions 1.0 - 1.9. Developed by [Pure Storage, Inc](http://www.purestorage.com/). Documentations can be found at [purity-fb.readthedocs.io](http://purity-fb.readthedocs.io/).

Reply:
ehh you probably don't need to update all of them. I just figured the setup and top level stuff was more likely to be read